lib/node_modules/@stdlib/stats/base/snanmeanpn/src/addon.c

```diff
@@ -34,10 +35,27 @@
 static napi_value addon( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV( env, info, argv, argc, 3 );
 	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
-	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 2 );
-	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, stride, argv, 1 );
-	STDLIB_NAPI_CREATE_DOUBLE( env, (double)stdlib_strided_snanmeanpn( N, X, stride ), v );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 2 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, strideX, argv, 1 );
+	STDLIB_NAPI_CREATE_DOUBLE( env, API_SUFFIX(stdlib_strided_snanmeanpn)( N, X, strideX ), v );
```

Comment:
```suggestion
	STDLIB_NAPI_CREATE_DOUBLE( env, (double)API_SUFFIX(stdlib_strided_snanmeanpn)( N, X, strideX ), v );
```

The return value is a `float`, so we need/want an explicit cast.